app/v2/notifications/post_notifications.py

```diff
@@ -169,6 +169,7 @@ def post_notification(notification_type):
             reply_to_text=reply_to
         )
 
+        # Think this is redundant
         template_with_content.values = notification.personalisation
```

Comment:
Shall we try removing it?

Reply:
It’s not redundant, it’s updating the personalisation to include any document download links. Will remove my comment.

Reply:
Done